widgets/userManagement/src/UsersTable.jsx

```diff
@@ -9,6 +9,8 @@ import TenantModal from './TenantModal';
 import UserDetails from './UserDetails';
 import UserPropType from './props/UserPropType';
 
+const columnT = key => Stage.i18n.t(`widgets.userManagement.columns.${key}`);
```

Comment:
Opitonal:

I would name it `columnsT`, since the part of `en.json` it refers to is `.columns.`, not `.column.` :slightly_smiling_face: 

```suggestion
const columnsT = key => Stage.i18n.t(`widgets.userManagement.columns.${key}`);
```

Reply:
it returns the translation for a single column, maybe it could be better to keep `columnT`

Reply:
Ok, if you see it that way, sure, it's ok to leave it unchanged :slightly_smiling_face: